lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -230,43 +237,41 @@ def write(chunk)
       # Group the entries since we have to make one call per tag.
       grouped_entries = {}
       chunk.msgpack_each do |tag, *arr|
-        if !grouped_entries.has_key?(tag)
-          grouped_entries[tag] = []
-        end
+        grouped_entries[tag] = [] unless grouped_entries.key?(tag)
```

Comment:
same general comment as before.. it's certainly unfamiliar to C++ types like us but appears to be idiomatic in Ruby...